Grundgeruest/admin.py

```diff
@@ -9,8 +9,31 @@
 # Register your models here.
 
 
+class StufenPlusUnterstuetzerListFilter(admin.SimpleListFilter):
+    """
+    This filter extends the common list_filter = ('stufe') by the option
+    "Unterstützer (alle)", i.e. stufe > 0.
+    """
+    title = 'Stufe'
+    parameter_name = 'stufe'
+
+    extended_stufe_choices = ScholariumProfile.stufe_choices.copy()
+    extended_stufe_choices.insert(1, ('unterstuetzer', 'Unterstützer (alle)'))
+
+    def lookups(self, request, model_admin):
+        return self.extended_stufe_choices
+
+    def queryset(self, request, queryset):
+        if self.value() == 'unterstuetzer':
+            return queryset.filter(stufe__gte=1)
+        else:
+            for choice in self.extended_stufe_choices:
+                if self.value() == str(choice[0]):
```

Comment:
more pythonic: tuple unpacking!
for key, value in ...:
    ... = str(key)

Reply:
Habe es geändert (und einen Rebase mit Squash gemacht).